src/pykeen/training/training_loop.py

```diff
@@ -120,6 +121,9 @@ def _get_lr_scheduler_kwargs(lr_scheduler: LRScheduler) -> Mapping[str, Any]:
     return lr_scheduler_kwargs
 
 
+worker_init_fn_resolver = FunctionResolver(elements=[literale_worker_init])
```

Comment:
I'm generally opposed to adding functionalities like this without a second or more potential elements in the resolver. If it can't be demonstrated any other ways to do this besides `literale_worker_init`, then why is it given as an option at all?

Reply:
I understand. So far, I know that also ConvE uses the same worker initialization, but I don't have an example of a model that requires a different one. However, this option potentially allows for accurate reproducibility of models, as one can e.g. set worker-specific random seeds.

How would you recommend integrating LiteralE's worker-specific random seeds?